### DISCUSSION TITLE
UMINHO/eeng PhD template

### DISCUSSION THREAD

**Discussion by DanielaOliveira-algoritmi** [joaomlourenco/novathesis#252] [Q&A] [answered]
Hello everyone,

I submitted my PhD thesis to the Uminho School of Engineering and it came back because I didn't have a complete cover (back + spine + front concatenated into a complete cover) and that is a problem.

Can anyone help me with this issue?

Thank you all!

**Comment by b-pereira** [comment, marked as answer]
Many colleagues didn't care about the spine and nobody told us about that —  until someone like you _test the waters_ and have to have that stress. We are really sorry for you for that. @DanielaOliveira-algoritmi  I'll ask @joaomlourenco to specify how to do that. However, it was just because of the lack of the spine, or really the full cover? **Maybe, most importantly, how urgent is this?**

EDIT: I asked and the maintainer said that there isn't any way to do that. In the past, something akin to that came about in a discussion and we talked about covers. Hence the idea that that was possible. _The question that arises is **exactly why** that  Uminho School of Engineering **rejected it**? Because you didn't **exactly** have a **complete cover (back + spine + front concatenated into a complete cover)**, or was it just **exactly the lack of the spine**?_

Then, we can discuss it more accurately and see what could be done.

**Reply by DanielaOliveira-algoritmi** [reply]
In the picture below, the current version of my thesis is on the left side, but it should be in the format of the example on the right side.

![Captura de ecrã 2021-11-15, às 19 57 15](https://user-images.githubusercontent.com/63778726/141846445-7a7b23b4-a483-48a5-9031-ebdfe6ca266d.png)

Thank you for your support, I will have to respond as soon as possible to Uminho eeng.

**Reply by b-pereira** [reply]
@DanielaOliveira-algoritmi check the edit that I made in the previous reply. They mandated **exactly that** or was it just because of the spine?

**Reply by DanielaOliveira-algoritmi** [reply]
They said exactly this: _"Devolução do processo: A tese não cumpre o estipulado no Despacho RT31/2019. Ponto 3. em falta a capa , a contra capa e a lombada da tese."_

**Reply by b-pereira** [reply]
@DanielaOliveira-algoritmi after looking carefully at your image, I found out another problem (maybe the **real problem**)
![imagem](https://user-images.githubusercontent.com/944511/141853189-80d6ed6d-8faf-4e0a-91ad-a02706602b16.png)

The following page after your cover has the front page. So I think that I am directly responsible, because while **main version** of this template **has everything**, mine, that is the one on https://github.com/b-pereira/novathesis, doesn't even support the spine. 

**In order to help you send a private message ASAP to https://www.facebook.com/pereirabruno05/**, I'll see that through with you. I am directly to blame for that.

**Reply by b-pereira** [reply]
This problem was related to the https://github.com/b-pereira/novathesis version of this template, namely, the one based on the 4.2.1 version of the official template. However, the problem is not specific only to that template's version, but with the cover and back cover overall. **The spine is not correct** in NOVAThesis official template and the cover and back-cover elements, regarding PhD theses — in which the printed version is **mandatory** — are not what the Teaching and Research Organic Unit requested, according to the image below, on the right. That image was sent by the School of Engineering specifying that **_the cover and back cover should be like that_.**

![thesis-covers](https://user-images.githubusercontent.com/63778726/141846445-7a7b23b4-a483-48a5-9031-ebdfe6ca266d.png)

However, a solution was found and will be posted in that specific version repository. A "feature request" for corrections will be created for the official template's version. **Note that** that the elements as they are in this official version are not to what the Teaching and Research Organic Units are used to. therefore it isn't known to what degree **the elements _as they are in the official template_ could be accepted**.

Many thanks to @DanielaOliveira-algoritmi for using the template for her PhD.
